### PR TITLE
Rewrite v1

### PR DESCRIPTION
**Pull request by shortdudey123**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
 # sc-mongodb Cookbook CHANGELOG
 
+## Unreleased
+
+## v1.0.0 Changes (Released 2017-05-23)
+
+*WARNING:* This is a rewrite the contains many backwards incompatable changes.  Many attributes have changed defaults and/or the attribute key itself.
```
Comment (rshade) [inline CHANGELOG.md]: this is a rewrite `that`

Comment (shortdudey123) [inline CHANGELOG.md]: fixed

**Comment by damacus** [comment]
Squash? :D :D

**Comment by shortdudey123** [comment]
i thought about it, but was thinking it might be good to not do it since it shows the flow the rewrite and its easier to pick though for people that want to read it

**Comment by damacus** [comment]
Sure! :shipit:

**Comment by shortdudey123** [comment]
@josephholsten can you review as well?

**Pull request merged**